MeshLib/Elements/Face.h

```diff
@@ -32,25 +30,10 @@ class Face : public Element
 {
 public:
 	/// Constant: Dimension of this mesh element
-	static const unsigned dimension = 2;
+	static const unsigned dimension = 2u;
 
 	/// Get the area of this 2d element.
-	virtual double getArea() const { return _area; }
-
-	/// Returns the length, area or volume of a 1D, 2D or 3D element
-	double getContent() const { return _area; }
-
-	/// Get dimension of the mesh element.
-	unsigned getDimension() const { return dimension; }
-
-	/// Returns the face i of the element.
-	const Element* getFace(unsigned i) const { return this->getEdge(i); }
-
-	/// Get the number of nodes for face i.
-	unsigned getNFaceNodes(unsigned i) const { (void)i; return 2; }
-
-	/// 2D elements have no faces.
-	unsigned getNFaces() const { return 0; }
+	virtual double getArea() const { return _content; }
```

Comment:
yes, ```getArea``` is not used ... it is one of the methods we included when we first implemented the MeshLib but in reality we always end up using ```getContent()``` instead ...